app/src/app/initApp.js

```diff
@@ -72,19 +73,27 @@ export default () => {
       }
     )
 
+    // workaround to solve problem:
+    // 1. move grape window to second screen
+    // 2. quit grape -> x or y might be negative now in windw-state.json
+    // 3. unplug monitor
+    // 4. start Grape again
+    // see GRAPE-14546
+    if (state.dimensions.x < 0 || state.dimensions.y < 0) {
+      state.prefs.x = 0;
+      state.prefs.y = 0;
+    }
+
     // set global to be accessible from webpage
     global.isNotificationSupported = isNotificationSupported()
     global.host = state.host
     global.grapeHost = env.host
     global.chooseDomainDisabled = env.chooseDomainDisabled
 
     state.mainWindow = new BrowserWindow(state.prefs)
-    if (state.dimensions.isMaximized && state.prefs.show) {
+    /* if (state.dimensions.isMaximized && state.prefs.show) {
```

Comment:
why is this commented out?Can it  be removed?

Reply:
will reactivate, turns out this is needeed to handle maximized windows. I had the wrong impression that electron-window-state handles that automatically